fab/operations/supervisor.py

```diff
@@ -277,10 +280,49 @@ def restart_all_except_webworkers():
     _services_restart()
 
 
+@roles(ROLES_STATIC)
+def _decommission_host(host):
+    files.comment(
+        '/etc/nginx/sites-available/{}_commcare'.format(env.environment),
+        '^[ ]*server {}'.format(host),
+        use_sudo=True,
+    )
+    _check_and_reload_nginx()
+
+
+@roles(ROLES_STATIC)
+def _recommission_host(host):
+    files.uncomment(
+        '/etc/nginx/sites-available/{}_commcare'.format(env.environment),
+        host,
```

Comment:
was more concerned about commenting out a line multiple times like `# # server host`, uncommenting doesn't have those issues so didn't think it was really necessary. can add something like `server {}.format(host)` though